server/py/services/api/tests/unit/utils/singletons/test_k8s_utils.py

```diff
@@ -279,6 +279,27 @@ def test_store_secret(
         assert data == secrets_to_store
 
 
+
+def test_store_secrets_does_not_pass_labels_to_read_secret(k8s_helper):
```

Comment:
test name too long. if you write in the comment of the test what it does, then IMO it can be shortened, something like:
```suggestion
def test_no_labels_in_read_secret(k8s_helper):
```

Reply:
or `test_store_secrets_ignores_labels`